### DISCUSSION TITLE
Source column not in TOM Explorer anymore

### DISCUSSION THREAD

**Discussion by rhains** [TabularEditor/TabularEditor3#1003] [🙋♀️TE3 Q&A] [answered]
For some reason in TE2 and TE3 latest versions the column "source" is not available anymore.
Is there any chance to put it back? Is this something that I need to configure in some kind of preferences?
This was really useful if you have many data sources as you can see from which DS each table was imported from.

**Comment by otykier** [comment]
I believe the column is still there in TE2. In TE3 it's called "Expression". However, you're right that it's not showing anything for regular tables, where as previously it showed the name of the data source. We'll make sure that's fixed.

**Comment by rhains** [comment]
Awesome thanks Daniel. Also something I find out , in my pc when I expand a table I see only measures but not the columns with each definition, however my coworkers can see the columns on their computer. I wonder if that is a preference I can setup somewhere. Maybe, exploring the tool I disabled a preference but do not remember where. Richard HainsOn Sep 29, 2023, at 01:58, Daniel Otykier ***@***.***> wrote:﻿
I believe the column is still there in TE2. In TE3 it's called "Expression". However, you're right that it's not showing anything for regular tables, where as previously it showed the name of the data source. We'll make sure that's fixed.

**Comment by edhans** [comment, marked as answer]
I bet you have this button disabled. Press it and you should see the columns. 
<img width="302" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/45298358/d497677c-48a5-4797-9fa1-dc8dcc1653b5">

**Comment by rhains** [comment]
I was referring the columns of the tables. Not the columns of the Tom
Explorer view.

Richard

**Comment by DBojsen** [comment]
Hi @rhains 

The columns of the tables are shown in the TOM explorer. @edhans answer is the correct one. 
Did you try it?

**Reply by rhains** [reply]
Thank you. edhans, that was the click I had to do to show the columns.
